scripts/demos/clp/init.sh

```diff
@@ -1,31 +1,31 @@
 #!/usr/bin/env bash
 rm -rf ~/.sifnoded
-rm -rf ~/.sifnodecli
+rm -rf ~/.sifnoded
 rm -rf sifnode.log
 rm -rf testlog.log
 
 
 sifnoded init test --chain-id=sifchain
 
-sifnodecli config output json
-sifnodecli config indent true
-sifnodecli config trust-node true
-sifnodecli config chain-id sifchain
-sifnodecli config keyring-backend test
+sifnoded config output json
+sifnoded config indent true
+sifnoded config trust-node true
+sifnoded config chain-id sifchain
+sifnoded config keyring-backend test
```

Comment:
config command has been removed, so if you want this to work you'll have to add some of these args in the below commands i.e keyring